### DISCUSSION TITLE
Does vlan-tx gets configured automatically by default?

### DISCUSSION THREAD

**Discussion by mkbchandran** [lldpd/lldpd#645] [General]
Is it the default behavior? - vlan-tx is configured automatically starting version 1.0.5 version.

**Comment by vincentbernat** [comment]
No, it is not (and it cannot as it requires a VLAN ID).

**Reply by mkbchandran** [reply]
Thanks! I never configured vlan-tx in the first place, but the LLDP daemon is generating vlan packets with id "0". Also, unconfiguring vlan-tx works, after which the LLDP daemon is sending regular packets. Still curious, how this is even possible!

 01:80:c2:00:00:0e, ethertype 802.1Q (0x8100), length 167: vlan 0, p 0, ethertype LLDP (0x88cc), LLDP <==== Default behavior
 01:80:c2:00:00:0e, ethertype LLDP (0x88cc), length 149: LLDP <==== After I unconfigure vlan-tx

**Reply by vincentbernat** [reply]
This has been fixed in 1.0.13.
